.vscode/tasks.json

```diff
@@ -3,6 +3,20 @@
     // for the documentation about the tasks.json format
     "version": "2.0.0",
     "tasks": [
+        {
+            "label": "Autoreconf",
+            "type": "shell",
+            "command": "autoreconf -if",
+            "problemMatcher": []
+        },
+        {
+            "label": "Configure",
+            "type": "shell",
+            "command": "./configure --prefix=/opt/ats --enable-ccache  --enable-experimental-plugins --enable-example-plugins --enable-test-tools --enable-debug --enable-werror",
```

Comment:
unless root access is absolutely necessary, I would not set the prefix in /opt. It also has the inconvenient of being global. I don't know if it's possible to use the workspace variable in this string, but something like `${workspaceFolder}/target` might be better.

This is not a strong opinion, so feel free to not change anything.

Reply:
Id be +1 on some env variable for it if possible, we tend to generate everything to use /opt/trafficserver here, so having that be customizable would be good.  Or maybe just have an optional env var insertion that people could throw in stuff like "--prefix=/your/ats/location --with-openssl=/your/ssl/location", so you can get custom things you use into the build

Reply:
@ezelkow1 I am playing around with it to see if I can set an environment variable to override the default

Reply:
@ezelkow1 I added the environment variable ATS_VSCODE_CONFIGURE, so you can override configuration options.  It looks like the last option wins in autoconf.